tests/org.eclipse.ui.tests/Eclipse UI Tests/org/eclipse/ui/tests/e4/CloseAllHandlerTest.java

```diff
@@ -119,46 +124,46 @@ public void testCloseMixedEditorTypes() {
 		// verify the close all handler is enabled now (since dummy editor has been
 		// opened)
 		canExecute = handlerService.canExecute(parameterizedCommand);
-		Assert.assertTrue(canExecute);
+		assertTrue(canExecute);
 
 		// close all editors (dummy editor should close!)
 		dummyPart = partService.findPart(DUMMY_E4_PART_ID);
-		Assert.assertNotNull(dummyPart);
+		assertNotNull(dummyPart);
 		handlerService.executeHandler(parameterizedCommand);
 		dummyPart = partService.findPart(DUMMY_E4_PART_ID);
-		Assert.assertNull(dummyPart);
+		assertNull(dummyPart);
 
 		// verify the close all handler is *not* enabled now (since dummy editor has
 		// been closed)
 		canExecute = handlerService.canExecute(parameterizedCommand);
-		Assert.assertFalse(canExecute);
+		assertFalse(canExecute);
 
 		// scenario 2: open a compatibility layer editor
 		IFileEditorInput input = new DummyFileEditorInput();
 		Object activeWindow = applicationContext.getActive(ISources.ACTIVE_WORKBENCH_WINDOW_NAME);
-		Assert.assertTrue("Active workbench window not found.", activeWindow instanceof IWorkbenchWindow);
+		assertTrue("Active workbench window not found.", activeWindow instanceof IWorkbenchWindow);
 		IWorkbenchWindow window = (IWorkbenchWindow) activeWindow;
```

Comment:
Probably the following would be possible, but not everyone finds it more readable.
```suggestion
		assertTrue("Active workbench window not found.", activeWindow instanceof IWorkbenchWindow window);
```

Reply:
Unfortunately this is not possible as that way window variable is local to the assertTrue call and thus not resolved outside of it.